### PR TITLE
Deploy EDS with GH actions

### PR DESCRIPTION
**Pull request by kittyflat**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+      - master
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install dependencies
+        run: yarn install --production=false
```
Comment (kittyflat) [inline .github/workflows/deploy.yml]: can try caching this

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      - name: Download fonts from S3
+        uses: actions/aws/cli@master
```
Comment (kittyflat) [inline .github/workflows/deploy.yml]: building this action takes > 1m but not sure how to cache this

**Comment by kittyflat** [comment]
still "room for improvement" with caching but it's also working now

File: .aws/config [hunk 3]
```diff
@@ -0,0 +1,4 @@
+[profile ci-user]
+role_arn = arn:aws:iam::248295656887:role/ci-user
```
Comment (roblevintennis) [inline .aws/config]: Should we be exposing `248295656887` here? This is a public repo

File: .github/workflows/deploy.yml [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - name: Download build files
+        uses: actions/download-artifact@master
+        with:
+          name: styleguide
+      - name: Sync styleguide to S3
+        uses: actions/aws/cli@master
+        with:
+          args: s3 sync ${GITHUB_WORKSPACE}/styleguide s3://eds.ethoslabs.io/ --delete --exclude "fonts.zip" --profile ci-user
```
Comment (roblevintennis) [inline .github/workflows/deploy.yml]: Does `sync` and `delete` here basically equate to an upsert where we're overwriting with new styleguide correct?

File: .aws/config (hunk 3 above)
Comment (kittyflat) [reply .aws/config]: good point, let me see if there's some way to remove it from the repo

File: .github/workflows/deploy.yml (hunk 4 above)
Comment (kittyflat) [reply .github/workflows/deploy.yml]: yeah `--delete` means delete everything that doesn't match the source, `--exclude` except for "fonts.zip"

**Pull request merged**